### PR TITLE
feat(circuits) - shift from circom_helper to circom_tester

### PR DESCRIPTION
**Pull request by ctrlc03**
Currently, we are using https://github.com/weijiekoh/circom-helper, which has not been maintained since October 2022. On the other hand, [cicom_tester](https://github.com/iden3/circom_tester) is currently being worked on, and seems to be used by other projects to test circom circuits (see [rln](https://github.com/Rate-Limiting-Nullifier/circom-rln/blob/main/test/rln.test.ts)).

The setup of circom_tester is much shorter, and can also be used to inspect individual signals if needed.

### CODE REVIEW

**Comment by baumstern** [comment]
Great work! Could you just update [README.md](https://github.com/privacy-scaling-explorations/maci/blob/dev/circuits/README.md) of circuit package?

**Review by baumstern — APPROVED** [review]
LGTM!

File: circuits/ts/__tests__/Hasher.test.ts [hunk 1]
```diff
@@ -19,29 +18,44 @@ import {
     hash4,
     hash3,
 } from 'maci-crypto'
+import * as path from 'path'
+import { expect } from 'chai'
+
+describe('Poseidon hash circuits', function() {
+    this.timeout(30000)
```
Comment (samajammin) [inline circuits/ts/__tests__/Hasher.test.ts:25]: Why is this necessary?

File: circuits/ts/__tests__/Ecdh.test.ts [hunk 2]
```diff
@@ -1,20 +1,21 @@
-jest.setTimeout(120000)
```
Comment (samajammin) [inline circuits/ts/__tests__/Ecdh.test.ts:1]: 🧐

**Comment by samajammin** [comment]
Nice work @ctrlc03!

Potential future investigation - have you heard of circomkit? I saw someone mention it:
https://github.com/erhant/circomkit

File: circuits/ts/__tests__/Hasher.test.ts (hunk 1 above)
Comment (ctrlc03) [reply circuits/ts/__tests__/Hasher.test.ts:25]: testing frameworks have some default timeout (you can usually set it globally or per test file) - given the nature of these tests, they can take a long time to run and default timeouts might be triggered

File: circuits/ts/__tests__/Ecdh.test.ts (hunk 2 above)
Comment (ctrlc03) [reply circuits/ts/__tests__/Ecdh.test.ts:1]: as circom_tester uses mocha, any jest reference was removed

**Comment by ctrlc03** [comment]
> Nice work @ctrlc03!
> 
> Potential future investigation - have you heard of circomkit? I saw someone mention it: https://github.com/erhant/circomkit

Thanks for pointing to this, looks promising, will see how it evolves!!
